locations/spiders/loxam_fr.py

```diff
@@ -0,0 +1,20 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class LoxamFrSpider(SitemapSpider, StructuredDataSpider):
+    name = "loxam_fr"
+    item_attributes = {"brand": "Loxam", "brand_wikidata": "Q3264407"}
+    allowed_domains = ["agence.loxam.fr"]
+    sitemap_urls = [
+        "https://agence.loxam.fr/locationsitemap1.xml",
+        "https://agence.loxam.fr/locationsitemap2.xml",
+        "https://agence.loxam.fr/locationsitemap3.xml",
+    ]
```

Comment:
```suggestion
    sitemap_urls = ["https://agence.loxam.fr/robots.txt"]
    sitemap_follow = ["locationsitemap"]
```
This way we are safe when `locationsitemap4.xml` gets added.

Reply:
Oh. I did not know sitemap_follow was an option. Not many spiders have it. Thanks for the feedback!